src/main/java/com/mabl/integration/jenkins/domain/ExecutionResult.java

```diff
@@ -165,6 +164,7 @@ public PlanExecutionResult(
     public static final class EventStatus {
         public Boolean succeeded;
         public Boolean succeededFirstAttempt;
+        public Boolean succeededWithRetries;
```

Comment:
Is this redundant? If not succeededFirstAttempt that means we succeeded with retries, right?

Reply:
This is a really good question, but we expose this in our API! I think we need to ask the question there.

Reply:
Actually, the answer is no, I do not think this is redundant. You can have an event that failed to execute all plans successfully on *both* the first attempt and on the retry as well. Also, you may not set retry on the plan overrides when you schedule a deployment event (or in any of the triggered plans).